be/src/runtime/task_group/task_group.h

```diff
@@ -121,6 +129,15 @@ class TaskGroup : public std::enable_shared_from_this<TaskGroup> {
 
     int64_t gc_memory(int64_t need_free_mem, RuntimeProfile* profile);
 
+    void upsert_task_scheduler(taskgroup::TaskGroupInfo* tg_info, ExecEnv* exec_env);
+
+    void get_query_scheduler(doris::pipeline::TaskScheduler** exec_sched,
+                             vectorized::SimplifiedScanScheduler** scan_sched,
+                             ThreadPool** non_pipe_thread_pool,
+                             vectorized::SimplifiedScanScheduler** remote_scan_sched);
+
+    void try_stop_task_scheduler();
```

Comment:
rename to try_stop_schedulers. because task_scheduler is for Pipeline task scheduler. It is very ambiguous